packages/flutter_tools/lib/src/localizations/gen_l10n_types.dart

```diff
@@ -331,13 +335,23 @@ class Message {
       );
     }
 
-    final RegExpMatch? pluralRegExp = _pluralRE.firstMatch(_value(bundle, resourceId));
-    final bool isPlural = pluralRegExp != null && pluralRegExp.groupCount == 1;
-    if (attributes == null && isPlural) {
-      throw L10nException(
-        'Resource attribute "@$resourceId" was not found. Please '
-        'ensure that plural resources have a corresponding @resource.'
-      );
+    if (attributes == null) {
+
+      void _throwEmptyAttributes(final RegExp regExp, final String type) {
+        final RegExpMatch? match = regExp.firstMatch(
```

Comment:
Two-space indent and everything on one line (maybe you accidentally ran flutter format)?

Reply:
I am using Android Studio, and it automatically formats lines obviously.

What the IDE used by the team? Or what specific configurations to be used, since this is frequently happening.